.scripts/package.sh

```diff
@@ -260,13 +260,25 @@ generate_swift_package () {
     template_replace $package "// GENERATE BINARIES" $binaries; rm -f $binaries
 }
 
+login_default() {
+    echo "$GITHUB_TOKEN" | gh auth login --with-token
+}
+
+login_reviewer() {
+    echo "$REVIEWER_TOKEN" | gh auth login --with-token
+}
```

Comment:
Not sure I understand why we need both. Is it b/c the reviewer and merger can't be the same persona?

Reply:
Yeah, to get around the org-wide one reviewer required rule.